### PR TITLE
Feature/spec helper/simple cov

### PR DESCRIPTION
**Pull request by TLamb32**
## Description

Add simplecov functionality by editing spec_helper and including "require 'spec_helper'" across all test files.

## Type of change

- [ ] fix
- [x] feat
- [ ] test
- [ ] refactor
- [ ] docs

## Checklist

- [x] code has been self reviewed
- [x] code runs without any errors
- [x] thorough testing has been implemented if adding feature
- [x] all tests pass

### Thanks!

### CODE REVIEW

**Comment by TLamb32** [comment]
FYI: This PR has all stat_tracker_spec tests unskipped. Beware if running rspec as it takes nearly 3 minutes to run through all tests!

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -1,2 +1,11 @@
...
+SimpleCov.start
+
+require 'csv'
+require './lib/game'
+require './lib/team'
+require './lib/team_result'
+require './lib/game_parser'
+require './lib/league_parser'
+require './lib/team_result_parser'
+require './lib/stat_tracker'
```
Comment (ttakahashi1591) [inline spec/spec_helper.rb:11]: This looks great! Thanks for building this `spec_helper` out to help keep things more organized.

**Review by ttakahashi1591 — APPROVED** [review]
This all looks great. Thanks again for the getting all of this set up for us!

File: spec/spec_helper.rb
Comment (lanebret85) [inline spec/spec_helper.rb:1]: Love the spec helper! Makes things so much more concise!

**Review by lanebret85 — APPROVED** [review]
Thanks for getting simplecov going! (And not on main this time)

**Pull request merged**